app/Libraries/Elasticsearch/Search.php

```diff
@@ -257,14 +259,21 @@ private function getDatadogTags()
 
     private function handleError(ElasticsearchException $e, string $operation)
     {
+        $err = json_decode($e->getMessage(), true);
+
+        if (is_array($err) && str_starts_with($err['error']['caused_by']['reason'] ?? '', 'Failed to parse search_after value for field ')) {
+            $e = new InvalidCursorException();
```

Comment:
function signature says `ElasticsearchException` but php doesn't complain if it's set to something else 🤔